compose/extends.md

```diff
@@ -6,14 +6,14 @@ title: Share Compose configurations between files and projects
 
 Compose supports two methods of sharing common configuration:
 
-1. Extending an entire Compose file by
+1. Extend an entire Compose file by
    [using multiple Compose files](extends.md#multiple-compose-files)
-2. Extending individual services with [the `extends` field](extends.md#extending-services) (for Compose file versions up to 2.1)
+2. Extend individual services with [the `extends` field](extends.md#extending-services) (for Compose file versions up to 2.1)
```

Comment:
Maybe we could clarify that this option is no longer valid. Not sure I succeded but you catch my drift.

```suggestion
2. Extend individual services with [the `extends` field](extends.md#extending-services). This option is only usable in older Compose file versions, up to version 2.1.
```

Reply:
Oo actually this is wrong. It's available in the current spec as well. Have fixed